### ISSUE TITLE
L1 Loss Documentation Issue

### ISSUE THREAD

**Issue by justinkterry** [pytorch/pytorch#12751]
## 📚 Documentation

The documentation for torch.nn.L1Loss needs to say that it's the same as MAE loss super clearly in the documentation. Everyone knows what MAE is, and not nearly as many people know what L1 is. I've had a bunch of students come to my office hours with the documentation page open saying "MAE isn't in PyTorch, what's going on here?" That shouldn't be happening. Also, you kind of mix notations by calling MAE L1 and calling MSE MSE, not L2.

**Comment by Kaixhin** [comment]
I've put in a fix for this. Considering we can't rename widely used modules unfortunately, at least MAE should be returned when searching the docs. I've also noted that the MSE loss computes the squared L2 norm, so that also be flagged up in doc searches.

**Comment by justinkterry** [comment]
Thanks. Thoughts on creating alias modules (l2loss=mse and mae=l1loss)?

**Comment by Kaixhin** [comment]
Unfortunately this would lead to bloat as many things have different names, e.g., the smooth L1 loss in Torch is also known as the Huber loss, and there's several names for transposed convolutions. The recommendation would be to define those aliases in your own code if you prefer them.

**Comment by justinkterry** [comment]
This appears to not have been merged? What's going on here?

**Comment by SsnL** [comment]
It is merged. Unfortunately, our bot uses a way to merge PR that doesn’t
mark them as merged on GH.
